frameworks/javascript/cw-2.js

```diff
@@ -356,6 +356,12 @@ try {
         Test.expect(false, msg, options);
       }
     },
+    assertFuzzyEquals: function (actual, expected, msg = "") {
```

Comment:
Also can we comment the method to indicate its fuzzy for float values. I could see it easily being confused with `assertSimilar`